# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pfstmo/autoptr.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pfstmo-1.4/src/mantiuk08/display_adaptive_tmo.h_orig	2010-08-11 01:04:34.000000000 +0000
+++ pfstmo-1.4/src/mantiuk08/display_adaptive_tmo.h	2010-08-11 01:02:47.000000000 +0000
@@ -29,6 +29,8 @@
  * $Id: display_adaptive_tmo.h,v 1.12 2009/02/23 18:46:36 rafm Exp $
  */
 
+#include <memory>
+
 #include "display_function.h"
 #include "display_size.h"
 
